backend/app/routers/country.py

```diff
@@ -0,0 +1,51 @@
+import httpx
+from app.config import Environment
+from app.config import get_settings
+from fastapi import APIRouter
+from fastapi import Request
+from fastapi.responses import JSONResponse
+
+
+router = APIRouter(
+    prefix="/country",
+    tags=["country"],
+    responses={404: {"description": "Country not found"}},
+)
+
+
+@router.get("/")
+async def read_all_genres(request: Request):
```

Comment:
Am I missing something, or should this endpoint be named something else?

Reply:
Oh haha, you're not, I just forgot to rename it :D Should be `lookup_country()` or something like that